arduino-ide-extension/src/browser/library/library-list-widget.ts

```diff
@@ -201,7 +201,9 @@ class MessageBoxDialog extends AbstractDialog<MessageBoxDialog.Result> {
       options.buttons || [nls.localize('vscode/issueMainService/ok', 'OK')]
     ).forEach((text, index) => {
       const button = this.createButton(text);
-      button.classList.add(index === 0 ? 'main' : 'secondary');
+      const isPrimaryButton =
+        index === (options.buttons ? options.buttons?.length - 1 : 0);
```

Comment:
`options.buttons?.length` should be `options.buttons.length`

Reply:
You're right. Changed [here](https://github.com/arduino/arduino-ide/pull/1382/commits/769f08c359f3aa02e2555d6d5e8589168abf7f97).